### PR TITLE
Calculator UI component

### PR DESCRIPTION
**Pull request by marvin-nyalik**
In this branch:

- I added a Calculator component to act as a container for the buttons
- I added a Button component to render all buttons of the calculator
- Ensured no linter errors
- Used GitFlow

Let me know of any required changes

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import React from 'react';
```
Comment (GzimAsani) [inline src/index.js:1]: - [ ] All the boilerplate need to be removed because it is unnecessary
image

File: src/components/Calculator.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+.keys-container {
+  width: 100%;
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+}
+
+.calc-key {
+  width: 100%;
+}
```
Comment (GzimAsani) [inline src/components/Calculator.css]: - [ ] Please follow the design more closely:
- The results part does not have the same color as on design
- Please reduce the width of the calculator in order to match the design betterv
image
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @marvin-nyalik                           :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with creating the calculator component correctly :+1: 

 ## Required Changes 

- [ ] Check for the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @marvin-nyalik, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**